doc/script_commands.txt

```diff
@@ -2260,6 +2260,62 @@ array, shifting all the elements beyond this towards the beginning.
 
 ---------------------------------------
 
+*inarray <array name>,<value>;
+
+This command return the index of the first value found in the array
+It will return -1 if the value not found.
```

Comment:
`This command returns the index of the first matching value found in the array.`
`It will return -1 if the value is not found.`